manager/integration/tests/test_basic.py

```diff
@@ -549,6 +549,126 @@ def test_restore_inc(clients, core_api, volume_name, pod):  # NOQA
+    assert interval["value"] == "0"
+
+    volname = generate_volume_name()
+    volume = client.create_volume(name=volname, size=SIZE, numberOfReplicas=3)
+    lht_host_id = get_self_host_id()
+    volume = common.wait_for_volume_detached(client, volname)
+    assert volume["name"] == volname
+    assert volume["size"] == SIZE
+    assert volume["numberOfReplicas"] == 3
+    assert volume["state"] == "detached"
+    assert volume["restorationRequired"] is False
+
+    volume.attach(hostId=lht_host_id)
+    volume = common.wait_for_volume_healthy(client, volname)
+    bv, b, snap, data = create_backup(client, volname)
+
+    sb_volume_name = "sb-"+ volname
+    sb_volume_name2 = "sb2-" + volname
+    sb_volume_name3 = "sb3-" + volname
+    client.create_volume(name=sb_volume_name, size=SIZE,
+                         numberOfReplicas=2, fromBackup=b['url'],
+                         frontend="", standby=True)
+    client.create_volume(name=sb_volume_name2, size=SIZE,
+                         numberOfReplicas=2, fromBackup=b['url'],
+                         frontend="", standby=True)
+    client.create_volume(name=sb_volume_name3, size=SIZE,
+                         numberOfReplicas=2, fromBackup=b['url'],
+                         frontend="", standby=True)
+    sb_volume = common.wait_for_volume_detached(client, sb_volume_name)
+    sb_volume2 = common.wait_for_volume_detached(client, sb_volume_name2)
+    sb_volume3 = common.wait_for_volume_detached(client, sb_volume_name3)
+    sb_volume.attach(hostId=lht_host_id)
+    sb_volume2.attach(hostId=lht_host_id)
+    sb_volume3.attach(hostId=lht_host_id)
+    sb_volume = common.wait_for_volume_healthy(client, sb_volume_name)
+    sb_volume2 = common.wait_for_volume_healthy(client, sb_volume_name2)
+    sb_volume3 = common.wait_for_volume_healthy(client, sb_volume_name3)
+
+    for i in range(RETRY_COUNTS):
+        sb_volume = client.by_id_volume(sb_volume_name)
+        sb_volume2 = client.by_id_volume(sb_volume_name2)
+        sb_volume3 = client.by_id_volume(sb_volume_name3)
+        if sb_volume["lastBackup"] != b["name"] or \
+                sb_volume2["lastBackup"] != b["name"] or \
+                sb_volume3["lastBackup"] != b["name"]:
+            time.sleep(RETRY_INTERVAL)
+        else:
+            break
+    assert sb_volume["standby"] is True
+    assert sb_volume["lastBackup"] == b["name"]
+    assert sb_volume2["standby"] is True
+    assert sb_volume2["lastBackup"] == b["name"]
+    assert sb_volume3["standby"] is True
+    assert sb_volume3["lastBackup"] == b["name"]
+
+    activate_standby_volume(client, sb_volume_name)
+    sb_volume = client.by_id_volume(sb_volume_name)
+    sb_volume.attach(hostId=lht_host_id)
+    sb_volume = common.wait_for_volume_healthy(client, sb_volume_name)
+    check_volume_data(sb_volume, data, False)
+
+    # check if the standby volume data still remains the same with the first backup
+    # since backupstore polling has been disabled
```

Comment:
If the activation will call additional pulling, I also feel strange about why this can pass.....
I tested it, when we set polling interval to 0, and there is new backup that has not been polled, then we activate that DR volume, it will fail and say "incremental restore has not been finished so can't activate", and then pulling begins. Then we activate again, it will be successful. So I think setting interval to 0 doesn't affect the "activate" feature. And I will write test cases to make sure about that.